crypto/evp/e_aes.c

```diff
@@ -396,19 +396,9 @@ static int aesni_xts_init_key(EVP_CIPHER_CTX *ctx, const unsigned char *key,
 
         /*
          * Verify that the two keys are different.
-         *
-         * This addresses the vulnerability described in Rogaway's
-         * September 2004 paper:
-         *
-         *      "Efficient Instantiations of Tweakable Blockciphers and
-         *       Refinements to Modes OCB and PMAC".
-         *      (http://web.cs.ucdavis.edu/~rogaway/papers/offsets.pdf)
-         *
-         * FIPS 140-2 IG A.9 XTS-AES Key Generation Requirements states
-         * that:
-         *      "The check for Key_1 != Key_2 shall be done at any place
-         *       BEFORE using the keys in the XTS-AES algorithm to process
-         *       data with them."
+         * 
+         * This addresses Rogaway's vulnerability,
+         * See comment in aes_xts_init_key() below.
```

Comment:
'See' -> 'see'  (or replace ',' by '.' )

Reply:
I meant for these to be separate sentences, so I'll replace the comma.